bcauth/jinja2/oauth2_provider/application_confirm_delete.html

```diff
@@ -0,0 +1,18 @@
+{% extends "account/profile.html" %}
+
+{% block head_subtitle %} - {{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+{% block body_title %}{{ _("OAuth2 Client Application") }} - {{ _("Delete") }}{% endblock %}
+
+{% block content %}
+    <h3>{{ _("Are you sure to delete the application %(app_name)s?") | format(app_name=application.name) }}</h3>
```

Comment:
You should use Jinja's newstyle gettext.  So then this would be something like:

```
<h3>{{ _("Are you sure to delete the application %(app_name)s?", app_name=application.name) }}</h3>
```

I don't know if you're familiar with newstyle or not. If not, there's some details about newstyle here:

http://jinja.pocoo.org/docs/dev/extensions/#newstyle-gettext